docs/shared-content/structured-configuration-variables.include.md

```diff
@@ -222,157 +222,132 @@ foo:
-XPath can also select comments and replace them if that's a requirement. Using the Octopus Variable named `/configuration/comment()` and the value of `Replaced Comment` you can replace the comment in the following xml:
-
-**XML Structure with comment**
-```xml
-<configuration>
-   <!-- Comment -->
-</configuration>
-```
-
-**XML Structure with comment replaced**
 ```xml
 <configuration>
-   <!-- Replaced Comment -->
+  <email role="admin">chief@example.org</email>
+  <email role="user">user@example.com</email>
 </configuration>
 ```
 
-### Processing Instructions
+### XML CDATA sections
 
-Processing Instructions can be replaced with the XPath processing instruction selector like so: `/document/processing-instruction('xml-stylesheet')`. When replacing a processing instruction, it's not possible to replace the individual attributes. The whole processing instruction gets replaced with the supplied value. Take the following example:
+CDATA sections can be replaced just like any other node by selecting them with the XPath. When the content of the CDATA section is replaced, the CDATA presentation is maintained in the output. In the following example, `development` in the CDATA tag can be replaced with `prod<1>` by having a variable `/document/environment/text()` with the value `prod<1>`:
 
-**XML Structure Processing Instruction**
+**XML Structure with CDATA**
 ```xml
 <document>
-   <?xml-stylesheet type="text/xsl" href="/Content/Glossary/main.xsl"?>
+    <environment><![CDATA[development]]></environment>
 </document>
 ```
 
-When the Octopus Variable `/document/processing-instruction('xml-stylesheet')` is set to `new value` the output will be the following:
-
-**XML Structure Processing Instruction replaced**
+**XML Structure with CDATA Replaced**
 ```xml
 <document>
-   <?xml-stylesheet new-value ?>
+  <environment><![CDATA[prod<1>]]></environment>
 </document>
 ```
 
 ### Namespaces
 
-When parsing the xml document, a namespace manager is built up first with all the local names and values from the defined namespaces. A limitation of namespaces in Octopus is that it's not possible to define the same namespace twice within a single XML document. Doing so will result in a warning being logged during a deployment and possible unexpected behavior if trying to use XPaths with namespaces. An example of a warning is as follows:
+When parsing the XML document, Octopus collects all namespace declarations for use in XPath expressions, so you can use any of the declared prefixes.
 
-```
-The namespace 'http://octopus.com' could not be mapped to the 'octopus' prefix, as another namespace 'http://octopus.com/xml' is already mapped to that prefix. XPath selectors using this prefix may not return the expected nodes. You can avoid this by ensuring all namespaces in your document have unique prefixes.
-```
\ No newline at end of file
+One limitation is that if the same prefix is declared more than once, only the first will be available in XPath expressions.
```

Comment:
- Here I think we can more succinctly explain why the namespaces are useful: it's so that you can use them in your XPaths. (With many XPath tools you have to declare them again alongside your XPath.)
- I'd reframe the limitation with duplicates; it just makes XPaths a bit ambiguous. It's so unlikely that I think one sentence is enough; the user will get the warning if it happens.